### PR TITLE
Add configuration version.major_minor to  jcryptool.rb

### PR DESCRIPTION
**Pull request by suschizu**
just 0.9 to find on the website

### CODE REVIEW

**Comment by ran-dall** [comment]
I'm getting a page with `Error 404`

File: Casks/jcryptool.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
   url "https://www.cryptool.org/jctdownload/Builds/downloads/stable/jcryptool-#{version}-macosx.cocoa.x86_64.tar.gz"
   appcast 'https://www.cryptool.org/en/jct-downloads/stable'
```
Comment (miccal) [inline Casks/jcryptool.rb:6]:
```
appcast 'https://www.cryptool.org/en/jct-downloads',
```
Needs a comma `,` after `appcast`.

Comment (ran-dall) [reply Casks/jcryptool.rb:6]: @miccal You're not getting a 404 page?

Screen Shot 2020-05-26 at 10 00 03 PM

Comment (miccal) [reply Casks/jcryptool.rb:6]: @ran-dall no, not from `https://www.cryptool.org/en/jct-downloads` but I am from the original `appcast` at `https://www.cryptool.org/en/jct-downloads/stable`:

Screen Shot 2020-05-27 at 13 20 23

Comment (ran-dall) [reply Casks/jcryptool.rb:6]: @miccal  Oh, Ok, I get it what you're saying.

**Pull request merged**